challenges/odd-divisor/solutions/bens-first-attempt.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Ben's First Attempt
+ * Author: Ben Pearson
+ */
+
+function isEven(input) {
+  return input % 2 === 0;
+}
+
+function halfUntilOdd(input) {
+  const half = input / 2;
```

Comment:
May be try `const half = input * 0.5` would be a little bit faster?
https://stackoverflow.com/questions/226465/should-i-use-multiplication-or-division